src/main/java/uk/gov/hmcts/cmc/claimstore/events/response/DefendantResponseCitizenNotificationsHandler.java

```diff
@@ -41,8 +41,8 @@ private boolean isAdmissionResponse(Claim claim) {
     @EventListener
     public void notifyClaimantResponse(DefendantResponseEvent event) {
         Claim claim = event.getClaim();
-        if (isAdmissionResponse(claim)) {
-            return;
+        if (!claim.getResponse().isPresent()) {
```

Comment:
This looks unnecessary to me. Even if you really want to add this check, there is no check for enclosing object (`claim`). 

Also prefer `IllegalArgumentException` instead of `IllegalStateException` when checking valid method arguments.
You may also consider using `Objects.requireNonNull(object, "error message")` if looking for IDE warnings etc.

Reply:
Added check for claim and changed exception type to IllegalArgumentException